sdk/storage/azure_storage_blob/tests/test_blob_client.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use azure_core_test::{recorded, TestContext};
+use azure_identity::DefaultAzureCredentialBuilder;
+use azure_storage_blob::blob_client::BlobClientOptions;
+use azure_storage_blob::clients::BlobClient;
+use azure_storage_blob::models::blob_properties::{build_from_response_headers, BlobProperties};
+use azure_storage_blob::BlobBlobClientGetPropertiesOptions;
+use std::error::Error;
+
+#[cfg(test)]
+mod tests {
+
+    use azure_storage_blob::models::blob_properties;
+
+    use super::*;
+
+    #[recorded::test(live)]
```

Comment:
Note: This causes tests to be skipped (live tests skipped by default) when running the usual `cargo test`

Reply:
Yes. That's by design and documented because they are live-only tests. You need to pass `AZURE_TEST_MODE=live cargo test` instead (bash example).